commnad_line_rock_paper_scissors.py

```diff
@@ -0,0 +1,36 @@
+""" Object-Oriented Rock Paper Scissors """
+
```

Comment:
Blank line should be removed here, and [no space](https://www.python.org/dev/peps/pep-0257/#one-line-docstrings) should be separating the triple-quotes from your text:
```python
"""Object-oriented Rock Paper Scissors"""
import random
```

NB: We'll eventually circle back and focus more on conventions around writing docstrings